downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/CreateReleaseTarball.java

```diff
@@ -0,0 +1,87 @@
+package org.reactome.release.downloadDirectory;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Paths;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.api.errors.GitAPIException;
+
+public class CreateReleaseTarball {
+	private static final Logger logger = LogManager.getLogger();
+	
+	public static void execute(int releaseNumber) throws IOException, InterruptedException {
+		logger.info("Running CreateReleaseTarball");
+		String tarDirRelease = "reactome_tar/" + releaseNumber + "/Release";
+		String absReleaseDir = "/usr/local/gkb/scripts/release/website_files_update";
+		String absReactomeDir = "/usr/local/reactomes/Reactome/production/Website/static/download/" + releaseNumber;
+		Process makeReleaseDir = Runtime.getRuntime().exec("mkdir -p " + tarDirRelease);
+		makeReleaseDir.waitFor();
+		// Remove any existing Release repositorys
+		File releaseDir = new File(tarDirRelease);
+		if (releaseDir.list().length > 0) {
+			Process removeReleaseRepo = Runtime.getRuntime().exec("rm -r " + tarDirRelease);
+			removeReleaseRepo.waitFor();
+		} 
+
+		logger.info("Cloning Release: https://github.com/reactome/Release.git...");
+		try {
+			Git.cloneRepository().setURI("https://github.com/reactome/Release.git").setDirectory(Paths.get(tarDirRelease).toFile()).call();
```

Comment:
`https://github.com/reactome/Release.git` is repeated twice in a few lines. Refactor to a class-scoped constant, or a config value in the properties file.